halo/app/start.go

```diff
@@ -218,10 +280,16 @@ func Start(ctx context.Context, cfg Config) (<-chan error, func(context.Context)
 		}
 		cmtNode.Wait()
 
+		if err := server.Shutdown(ctx); err != nil {
+			log.Error(ctx, "Server shutdown failed:%+v", err)
+		}
+
 		// Note that cometBFT doesn't shut down cleanly. It leaves a bunch of goroutines running...
 
 		if err := stopTracer(ctx); err != nil {
 			return errors.Wrap(err, "stop tracer")
+		} else {
+			log.Info(ctx, "HTTP server stopped")
```

Comment:
avoid unnecessary logs, this doesn't add much value

Reply:
Yep, I added this when I was debugging the failing test.